CONTRIBUTING.md

```diff
@@ -142,6 +142,18 @@ pip install -r requirements.txt  # install dependencies
 pip install -e .  # install the local version of Scribe-Data
 pre-commit install  # install pre-commit hooks
 # pre-commit run --all-files  # lint and fix common problems in the codebase
+```
+ - On Windows run: 
```

Comment:
I think this section should be placed after this section, otherwise it might get confusing why commands for windows is in two placed. 

https://github.com/scribe-org/Scribe-Data/blob/64eb52b0be2d4fe7d9f212d733c602463c79046d/CONTRIBUTING.md?plain=1#L130-L135